.github/knip.ts

```diff
@@ -6,7 +6,7 @@ const config: KnipConfig = {
   ignore: ["src/types/config.ts", "**/__mocks__/**", "**/__fixtures__/**", "src/worker.ts"],
   ignoreExportsUsedInFile: true,
   // eslint can also be safely ignored as per the docs: https://knip.dev/guides/handling-issues#eslint--jest
-  ignoreDependencies: ["eslint-config-prettier", "eslint-plugin-prettier", "@types/jest"],
+  ignoreDependencies: ["eslint-config-prettier", "eslint-plugin-prettier", "@types/jest", "@actions/core", "@actions/github"],
```

Comment:
@Keyrxng So in the end are these needed?

Reply:
@gentlementlegen https://github.com/ubiquibot/command-start-stop/actions/runs/10077586980/job/27860439377?pr=10

Pretty sure they are needed yeah otherwise we'd see this warning in the above run log

> Unused item in ignoreDependencies: @actions/core
> Unused item in ignoreDependencies: @actions/github

Reply:
@Keyrxng Yes but these dependencies are actually never imported anywhere in the codebase from what I see, so Knip message makes sense, they can be removed from `package.json`.

Reply:
Ah I was on auto-pilot thinking they were in the action entry but it's been removed